boards/arm/bl654_dvk/bl654_dvk.dts

```diff
@@ -161,11 +161,21 @@
 		};
 		scratch_partition: partition@de000 {
 			label = "image-scratch";
-			reg = <0x000de000 0x0001e000>;
+			reg = <0x000de000 0x00012000>;
 		};
-		storage_partition: partition@fc000 {
+
+		/*
+		 * The flash starting at 0x000f0000 and ending at
+		 * 0x000fffff is reserved for use by the application.
+		 */
+
+		/*
+		 * If enabled, partition for FCB/NFFS will be created in this
```

Comment:
I think NVS also uses the storage partition, please add it to the list

Reply:
Added NVS to the list for all files.